.github/workflows/main.yml

```diff
@@ -130,5 +130,54 @@ jobs:
         with:
          name: app reports
          path: app/build/test-results
-      
-      
+
+  bazel_tests:
+    name: Bazel tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: ~/oppia-bazel/bazel-bin/src/bazel
+          key: ${{ runner.os }}-bazel-${{ hashFiles('**/*.bazel') }}}
+      - name: Setup bazel
+        uses: jwlawson/actions-setup-bazel@v1
+        with:
+          bazel-version: '3.4.1'
+      - name: Use bazel
+        run: bazel --version
+      - name: Clone Oppia Bazel
+        run: git clone https://github.com/miaboloix/bazel.git ~/oppia-bazel
```

Comment:
Can you send me brief instructions for how you set this up & any files I need to add so that I can create a version of it under https://github.com/oppia ?

Reply:
All I did was fork the bazel repo and apply the databinding patch (let me know if you need me to send you the exact file I used). I then uploaded the bazel-build.zip and bazel-tools.zip files. Let me know what the best way to send you these files is.

Reply:
Are we planning to clone in every build or can we store this in the repo?